screens/Issuers/IssuersScreen.tsx

```diff
@@ -124,24 +129,56 @@ export const IssuersScreen: React.FC<
     );
   }
 
+  const showAllIssuers = () => {
+    setFilteredSearchData(issuers);
+  };
+
+  const searchIssuer = text => {
+    if (text === '') {
+      setFilteredSearchData(issuers);
+      setSearch(text);
+    } else {
+      let filterdData = issuers.filter(item => {
+        if (
+          getDisplayObjectForCurrentLanguage(item.display)
+            ?.name.toLowerCase()
+            .includes(text.toLowerCase())
+        ) {
+          return getDisplayObjectForCurrentLanguage(item.display);
+        }
+      }, []);
+
+      setFilteredSearchData(filterdData);
+      setSearch(text);
+    }
+  };
+
   return (
     <React.Fragment>
       {controller.issuers.length > 0 && (
-        <Column style={Theme.issuersScreenStyles.issuerListOuterContainer}>
-          <Text
-            {...testIDProps('addCardDescription')}
-            style={{
-              ...Theme.TextStyles.regularGrey,
-              paddingTop: 0.5,
-              marginVertical: 14,
-              marginHorizontal: 9,
-            }}>
-            {t('header')}
-          </Text>
-          <View style={Theme.issuersScreenStyles.issuersContainer}>
+        <Column style={Theme.IssuersScreenStyles.issuerListOuterContainer}>
+          <Row margin="3">
+            <Icon
+              testID="search"
```

Comment:
can we use unique name for testId, like searchIssuer. 
because in future we might introduce search for vc's in homescereen as well.